### PR TITLE
Add note about Qt scaling (boo#1091103)

### PR DESCRIPTION
**Pull request by sknorr**
@Vogtinator Can you check that this still makes sense?

I am reasonably unsure about a number of things here... commented.

### CODE REVIEW

File: xml/release-notes.xml [hunk 1]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    On other desktops, this behavior of Qt can lead to undesirable situations
+    such as the following: If the primary display is High-DPI
+    (&gt;= 144 DPI), fonts in Qt applications that request scaling, such as
+    VLC, are effectively scaled to half the desired size on all monitors.
+    Applications which do not request scaling, such as &yast; (with default
+    settings), are not affected and use the same DPI value on all monitors.
```
Comment (sknorr) [inline xml/release-notes.xml]: Do we mean all monitors or just those that have 96 DPI?

File: xml/release-notes.xml [hunk 2]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+     <screen>echo Xft.dpi:<replaceable>DPI_VALUE</replaceable> | xrdb -nocpp -merge</screen>
+     <para>
+      Replace <replaceable>DPI_VALUE</replaceable> with an appropriate DPI
+      value for the primary monitor.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Disable automatic scaling:
+      <command>export QT_AUTO_SCREEN_SCALE_FACTOR=0</command>
```
Comment (sknorr) [inline xml/release-notes.xml]: How is this helping? Won't we just have tiny fonts on the High-DPI monitor?

File: xml/release-notes.xml [hunk 3]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      GNOME will set DPI according to each monitor's reported scaling.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      KDE Plasma disables automatic scaling by default.
```
Comment (sknorr) [inline xml/release-notes.xml]: It seems like that is just workaround (3) again...

File: xml/release-notes.xml [hunk 4]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+     </para>
+    </listitem>
+   </itemizedlist>
+   <para>
+    On other desktops, this behavior of Qt can lead to undesirable situations
+    such as the following: If the primary display is High-DPI
+    (&gt;= 144 DPI), fonts in Qt applications that request scaling, such as
+    VLC, are effectively scaled to half the desired size on all monitors.
```
Comment (sknorr) [inline xml/release-notes.xml]: Is this a misunderstading of what 0.5x scale means?

File: xml/release-notes.xml [hunk 5]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+    monitor to derive font DPI for all other monitors.
+   </para>
+   <para>
+    Two widely used desktops will override this behavior of Qt, therefore this
+    note does not apply to them:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      GNOME will set DPI according to each monitor's reported scaling.
```
Comment (Vogtinator) [inline xml/release-notes.xml]: This is wrong. GNOME sets Xft.dpi to 96 * the manually configured scale (100%/200%/300%)

File: xml/release-notes.xml (hunk 3 above)
Comment (Vogtinator) [reply xml/release-notes.xml]: This is misleading. Plasma disables Qt's internal automatic scaling, but sets up a different mechanism.

File: xml/release-notes.xml (hunk 4 above)
Comment (Vogtinator) [reply xml/release-notes.xml]: No, this is exactly how it appears.

File: xml/release-notes.xml (hunk 1 above)
Comment (Vogtinator) [reply xml/release-notes.xml]: It has the same size on all monitors, so looks normal on 96DPI and smaller on HiDPI.

File: xml/release-notes.xml [hunk 6]
```diff
@@ -648,6 +648,70 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_SETTING</replaceable>
...
+     <para>
+      Use a monitor with a regular DPI value as the primary monitor.
+      Applications that request scaling are then scaled appropriately on the
+      High-DPI monitor.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Set an appropriate font DPI (<literal>Xft.dpi</literal>) with the
+      configuration utility of your desktop or the following command:
```
Comment (Vogtinator) [inline xml/release-notes.xml]: Maybe mention that this needs to be done on every login?

File: xml/release-notes.xml (hunk 2 above)
Comment (Vogtinator) [reply xml/release-notes.xml]: Yes. It'll fix the appearance on the primary 96 DPI monitor though. The Xft.dpi workaround should cover most cases already, so you could probably remove this.

File: xml/release-notes.xml [hunk 7]
```diff
@@ -664,12 +664,14 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_VALUE</replaceable>
    <itemizedlist>
     <listitem>
      <para>
-      GNOME will set DPI according to each monitor's reported scaling.
+      GNOME will set <literal>Xft.dpi</literal> to the configured multiple of
```
Comment (Vogtinator) [inline xml/release-notes.xml]: There is only a single global Xft.dpi.

File: xml/release-notes.xml [hunk 8]
```diff
@@ -664,12 +664,14 @@ ServerArguments=-nolisten tcp -dpi <replaceable>DPI_VALUE</replaceable>
...
+      96 DPI for each monitor.
      </para>
     </listitem>
     <listitem>
      <para>
-      KDE Plasma disables automatic scaling by default.
+      KDE Plasma disables the automatic scaling of Qt but uses an own scaling
```
Comment (Vogtinator) [inline xml/release-notes.xml]: It configues Qt manually, so does not rely on DPI autodetection. The scaling mechanism is still the same in Qt.

Comment (sknorr) [reply xml/release-notes.xml]: Thanks for all the clarifications! It should now be correct.

**Comment by sknorr** [comment]
Pushed & merged as 20bc5d5eef0fa4ef35608f35ca1d3f64fdcf4b3b.

**Pull request closed without merging**